codecov.yml

```diff
@@ -4,15 +4,13 @@ codecov:
     wait_for_ci: false
   require_ci_to_pass: false
 coverage:
-  range: 75..98
+  range: 90..98
```

Comment:
The `range` here defines for Codecov reports when the value should appear as red (below 90%), yellow (between 90% to 98%), or green (98% to 100%).